.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -61,6 +61,8 @@ export default class Playground extends React.Component<never, State> {
 * [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
 * [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
 * [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
+* [ ] Updated `README.md` with documentation changes
+* [ ] Tophatted documentation changes in the style guide
```

Comment:
This would be nice to include too but there's no easy way of doing this check right now. What if we told people to check in a MD previewer, do you think that's enough? We could also write a quick doc for how to tophat changes in the styleguide and link it here too

Reply:
> We could also write a quick doc for how to tophat changes in the styleguide and link it here too

Can do. I think testing the look in the style guide is important, since folks in lots of different roles need to do it.

Reply:
Sounds good, it should just be a matter of documenting the `build-consumer` command and telling people to set up `polaris-styleguide` as well 👍

Reply:
Added a new doc, copying over some info from the `Tophatting.md` doc and adding instructions.